tests/CommandTest.php

```diff
@@ -93,6 +93,7 @@ public function testRequiredOptions()
     /**
      * @expectedException \Equip\Command\CommandException
      * @expectedExceptionCode \Equip\Command\CommandException::MISSING_OPTION
+     * @expectedExceptionRegExp /required options not defined:/i
```

Comment:
Sure, I suppose... I figured adding a message that definitely comes from `missingOptions` would be enough.